### ISSUE TITLE
Release 0.7.3

### ISSUE THREAD

**Issue by MattHJensen** [open-source-economics/Tax-Calculator#1085]
Please edit the table below adding your significant contributions as they are merged to master.

The highest-numbered pull request included in 0.7.2 was #1082, which was merged into master on 29-Nov-2016.

**Significant contributions to Tax-Calculator release 0.7.3**

| Contributor | PR# | Description |
| :-- | :-- | :-- |
| Martin Holmer | #1081 | add ability to use an expression to specify a policy parameter |
| Martin Holmer | #1083 | expand scope of JSON reform file to include non-policy parameters |
| Martin Holmer | #1088 | fix capital-gains-reform bug reported by Cody Kallen |
| Martin Holmer | #1098 | add ability to conduct normative expected-utility analysis |
| Cody Kallen | #1104 | add ability to compute MTR with respect to charitable cash contribs |
| Zach Risher | #1092 | provide Pandas 0.19.1 compat by fixing dataframe.to_csv() usage |
| Zach Risher | #1094 | unify environment definition by removing requirements.txt |
| Matt Jensen | #1109 | reorganize current_law_policy.json and add section headers |
| Cody Kallen | #1084 | add dollar limit on itemized deductions |
| T.J. Alumbaugh | #1111 | add testing for Windows with Appveyor |

**Comment by zrisher** [comment]
Zach Risher - #1092 - provide Pandas 0.19.1 compat by fixing `dataframe.to_csv()` usage
Zach Risher - #1094 - unify environment definition by removing `requirements.txt`

**Comment by MattHJensen** [comment]
I am hoping to release 0.7.3 on Monday after #1113 is resolved.

**Comment by MattHJensen** [comment]
Thanks everyone. Released 0.7.3 https://github.com/open-source-economics/Tax-Calculator/releases/tag/0.7.3 

Please let me know if you'd like any changes to the notes.
